apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -75,6 +75,21 @@ namespace MWMechanics
             mState = wanderState;
             mIsWanderingManually = isManualWander;
         }
+
+        AiWanderStorage(const AiWanderStorage& storage)
```

Comment:
Why default copy constructor is not good enough?

Reply:
I do not know if all compilers generate a constructor, which work in the same way.